packages/components/components/elvis-tabs/src/react/elvia-tabs.tsx

```diff
@@ -43,6 +43,18 @@ export const Tabs: FC<TabsProps> = ({
   });
   const scrollPosition = useScrollPositionDetection(scrollContainerRef);
 
+  const lang = useLanguage();
+  /** returns a localized scroll direction label based on the given direction and the current language */
+  const getScrollLabel = (direction: 'left' | 'right'): string => {
+    if (direction === 'left') {
+      return lang === 'no' ? 'Scroll mot venstre' : 'Scroll left';
+    } else if (direction === 'right') {
+      return lang === 'no' ? 'Scroll mot høyre' : 'Scroll right';
```

Comment:
I suggest changing it to "skroll" in Norwegian (Ref https://naob.no/ordbok/skrolle_3)

Reply:
I can see why this needs to be changed because the screen reader could read this with an accent that we don't want. Changes have been made.